CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * It is now possible to specity configuration options on the command line
   with he new `--config-toml` global option.
 
+* `jj git` subcommands will prompt for credentials when required for
```

Comment:
I think this PR resolves #469, or at least most of it, so add `(#469)` (see how it's done for #250 below).